raiden/api/rest.py

```diff
@@ -84,8 +95,8 @@ def add_resource(self, resource_cls):
             resource_cls._route
         )
 
-    def run(self, port, debug=False):
-        app.run(port=port, debug=debug)
+    def run(self, port, **kwargs):
+        self.flask_app.run(port=port, **kwargs)
```

Comment:
Okay, so I skimmed over this and my comment was obviously wrong, `kwargs` may contain the `host` key and that will configure the local ip-address/interface.

So either:
- We force `host` to be `127.0.0.1` or to prohibit requests from a different ip address like in the other comment.
- We add a configuration for addresses allowed to make requests, preferably with something like [flask-cors](https://flask-cors.readthedocs.io/en/latest/) to solve CORS too (need to check if the extension only sets the http headers or if it also checks the request's ip address).